### ISSUE TITLE
Select focus state not visible enough

### ISSUE THREAD

**Issue by Aniort** [Orange-OpenSource/Orange-Boosted-Bootstrap#659]
hello,
threre's an issue when the focus enter a <select> hml form, the focus status indicator is just displayed by a light gray to black 2px border. Even if it's enough for other f'orm controls (because the flashing cursor appears) it's not the case for a select.
We should increase the focus visiblity by, for example, changing the backgroàund ceolor of' the select or anything else...

**Comment by yannickcornaille** [comment]
I applied this kind of indicator to an Orange project with a "Brand Orange" focus like this :

![Capture d’écran 2021-09-21 à 15 59 11](https://user-images.githubusercontent.com/13216887/134198798-fbe47daf-9144-4bac-a54d-dd803c039137.png)

Maybe we could do this just for `<select>` or even all form components.

**Comment by ffoodd** [comment]
FWIW `:focus-visible` does not apply to form elements, so using such strong focus indicator will make it visible both for mouse and keyboard users. Which is usually good, but still: that's why it's not the case for now.
